module/window_swallowing.lua

```diff
@@ -63,7 +56,8 @@ local function manage_clientspawn(c)
     if
         -- will search for "(parent_client.pid)" inside the parent_pid string
         ( tostring(parent_pid):find("("..tostring(parent_client.pid)..")") )
-        and check_parent(parent_client.class) and check_client(c.class)
+        and check_swallow(parent_client.class, parent_filter_list)
+        and check_swallow(c.class, child_filter_list)
```

Comment:
Hm, actually, I'm not sure whether the `check_swallow` function makes sense at all now...

If its only usage is here, it would be easier to test values with `is_in_table` here (?) (and it would prevent to test `not swallowing_filter` twice)

Reply:
moved things around quite some, swallowing filter is only checked once now.